grakn-engine/src/main/java/ai/grakn/engine/controller/api/ConceptController.java

```diff
@@ -104,4 +113,34 @@ private String getConcept(Response response, Keyspace keyspace, Function<GraknTx
             }
         }
     }
+
+    private String getTypes(Request request, Response response) throws JsonProcessingException {
+        Function<GraknTx, Stream<? extends ai.grakn.concept.Concept>> getter = (tx) -> {
+            Stream<? extends ai.grakn.concept.Concept> stream = tx.admin().getMetaEntityType().subs();
+            stream = Stream.concat(stream, tx.admin().getMetaRelationType().subs());
+            stream = Stream.concat(stream, tx.admin().getMetaAttributeType().subs());
+            return stream;
```

Comment:
What about `(tx) -> tx.admin().getMetaConcept().subs()`?

Reply:
Ah yes we can indeed! I forgot that we disconnected the meta role and meta rule